addons/web/static/tests/views/kanban/kanban_view_tests.js

```diff
@@ -6256,6 +6256,52 @@ QUnit.module("Views", (hooks) => {
         );
     });
 
+    QUnit.test("delete an empty column, then a column with records.", async (assert) => {
+        patchDialog((_cls, props) => props.confirm());
+        let firstLoad = true;
+        patchWithCleanup(MockServer.prototype, {
```

Comment:
It s not a good idea too patch the MockServer. In your case, you just need to pass a mockRPC to "makeView".